Java/JDI/jdi-uitest-web/src/main/java/com/epam/jdi/uitests/web/selenium/elements/complex/RadioButtons.java

```diff
@@ -39,4 +52,22 @@ public RadioButtons(By optionsNamesLocatorTemplate, By allOptionsNamesLocator) {
         super(optionsNamesLocatorTemplate, allOptionsNamesLocator);
     }
 
+
+    public static void setUp(BaseElement el, Field field) {
+        if (!fieldHasAnnotation(field, JRadioButtons.class, IRadioButtons.class))
+            return;
+        ((RadioButtons) el).setUp(field.getAnnotation(JRadioButtons.class));
+    }
+
+    public RadioButtons setUp(JRadioButtons jRadioButtons) {
+        By root = findByToBy(jRadioButtons.root());
+        if (root == null) root = findByToBy(jRadioButtons.jRoot());
+        setAvatar(root);
+
+        By allLabels = findByToBy(jRadioButtons.allLabels());
+        if(allLabels == null) allLabels = findByToBy(jRadioButtons.jAllLabelsLocator());
```

Comment:
if block should be in its own line. please surround "allLabels = ...." with curly braces.